includes/compat/class-wc-stripe-email-failed-refund.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Base class for the email sent to the admin and customers when a refund fails.
+ *
+ * @since 9.6.0
+ */
+abstract class WC_Stripe_Email_Failed_Refund extends WC_Email_Failed_Order {
+	/**
+	 * Returns the list of template parameters.
+	 *
+	 * @return array
+	 */
+	abstract public function get_template_params();
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->title   = __( 'Refund request failed', 'woocommerce-gateway-stripe' );
+		$this->heading = __( 'Refund request failed', 'woocommerce-gateway-stripe' );
+		$this->subject = __( '[{site_title}] Refund request failed for #{order_number}.', 'woocommerce-gateway-stripe' );
```

Comment:
I think we should simplify the language here to say the refund failed, especially seeing as this email will be going to users.
```suggestion
		$this->title   = __( 'Refund failed', 'woocommerce-gateway-stripe' );
		$this->heading = __( 'Refund failed', 'woocommerce-gateway-stripe' );
		$this->subject = __( '[{site_title}] Refund failed for #{order_number}.', 'woocommerce-gateway-stripe' );
```